internal/driver/glfw/menu_darwin.go

```diff
@@ -35,6 +35,8 @@ const char* test_NSMenuItem_title(const void*);
 import "C"
 
 var callbacks []func()
+var menuEnableCallbacks []func() bool
+var menuCheckedCallbacks []func() bool
```

Comment:
Hmm, I don't know if it's just me but there is something with function pointer slices that feels wrong. I wonder if there is a more "sane" approach?

Reply:
There was a discussion on the original PR I think, but a better approach was not found.
I could convert it to a map of id->func() if you like, not sure if that adds anything though,

Reply:
Hmm, I don't think a map would gain us anything. Could be a good idea to see what others think about this and/or if they have any other solutions.

Reply:
I would agree but this has been open 24 days now, so not sure who else is available to help.

Reply:
We have not had any better ideas, given that this was following the pattern above I think we can progress and come back to it if there are great suggestions later.

Reply:
I thought @fpabl0 had an idea about it? https://gophers.slack.com/archives/CK5U4BU87/p1623952193313600

Reply:
Thanks for the reminder, I think that I have applied the solution now.